apps/release-notes/composable/classifyRelease.ts

```diff
@@ -21,8 +21,8 @@ export async function classifyReleases (
     endCursor: ''
   }
   if (team === 'ENTITIES') {
-    myGhIds = await getBoard('Entities Team Space - new')
-  } else if (team === 'NAMETEAMSPACE') {
+    myGhIds = await getBoard('entities---david')
```

Comment:
these are the entity board names in zenhub:

<img width="611" alt="Screenshot 2024-02-22 at 11 19 01 AM" src="https://github.com/bcgov/sbc-producthub/assets/22973013/37f5bcd0-3b91-4296-8977-62046e156177">

Reply:
they got divided into 2 teams

Reply:
oh i see i lost some changes while pushing

Reply:
yes. and the team names are 'Entities - David' and 'Entities - Olga', not 'entitities---david'